### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Bestbynature**
## Features included in the pull request are as follows:
---
- [x] The boilerplate from Create React App (text, images, styles) was deleted.
- [x] A new directory called components was created.
- [x] A new Calculator.js file was created inside the components folder.
- [x] In Calculator.js, a React component that matches the design was created following the design alone (JSX and CSS)
- [x]  Functional components were used
- [x] At least one child component was created and nested
- [x]  The calculator was then imported and used in the top-level component: App.js
- [x] It renders very well in the browser without problems.

### CODE REVIEW

File: src/components/styles.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+.operations {
+  background-color: rgb(247, 143, 8);
+  border-top: gray;
+}
+
+.calculator {
+  width: 50%;
+
+  /* border: 0.5rem solid black; */
```
Comment (OyePriscilla) [inline src/components/styles.css]: - [x] Please kindly remove this commented line of code to make your code clean according to practices. 👍

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/App';
 
+// ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (OyePriscilla) [inline src/index.js]: - [x]  Please kindly remove this commented line of code to make your code clean according to practices. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Bestbynature 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implement React functional components ✔️ 
Remove all boilerplate that got created by the CRA ✔️ 

## Required Changes ♻️

- [ ] Please kindly fix the Live Demo broken link from your README file or remove the live demo header if you don't have any deployment at the moment. ℹ️ **Note: The same is applicable to the link added to Mathematics Magician header as well**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (Bestbynature) [reply src/index.js]: Thank you so much for your keen observation

File: src/components/styles.css (hunk 1 above)
Comment (Bestbynature) [reply src/components/styles.css]: 👍

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @Bestbynature, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :green_circle: 

- PR is professional :+1: 
- Descriptive README.md file :+1: 


## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**